l1-contracts/contracts/bridge/L1SharedBridge.sol

```diff
@@ -688,23 +736,39 @@ contract L1SharedBridge is IL1SharedBridge, ReentrancyGuard, Ownable2StepUpgrade
     function depositLegacyErc20Bridge(
         address _prevMsgSender,
         address _l2Receiver,
-        address _l1Token,
+        address _l1Asset,
         uint256 _amount,
         uint256 _l2TxGasLimit,
         uint256 _l2TxGasPerPubdataByte,
         address _refundRecipient
     ) external payable override onlyLegacyBridge nonReentrant whenNotPaused returns (bytes32 l2TxHash) {
         require(l2BridgeAddress[ERA_CHAIN_ID] != address(0), "ShB b. n dep");
-        require(_l1Token != L1_WETH_TOKEN, "ShB: WETH deposit not supported 2");
+        require(_l1Asset != L1_WETH_TOKEN, "ShB: WETH deposit not supported 2");
```

Comment:
In this particular method I would keep the name `token`, since it is not any sort of asset